module/Finna/src/Finna/View/Helper/Root/OrganisationsList.php

```diff
@@ -114,15 +125,33 @@ public function __construct(
      */
     public function __invoke()
     {
-        $cacheName = 'organisations_list_' . $this->locale;
-        $list = $this->cache->getItem($cacheName);
+        return $this->generateOrganisationsList();
+    }
 
-        if (!$list) {
+    /**
+     * Generates a list of current organisations.
+     *
+     * @return array
+     */
+    protected function generateOrganisationsList(): array
+    {
+        $cacheDir = $this->cacheManager->getCache('organisation-info')->getOptions()
+            ->getCacheDir();
+        $cacheFile = "$cacheDir/organisations_list_ $this->locale.json";
```

Comment:
Onko toi välilyöntä tarkoituksellinen? Voisi olla selkeämp muodossa 
`        $cacheFile = "$cacheDir/organisations_list_{$this->locale}.json";`

Reply:
ei ole tarkoituksellinen...